app/lib/methods/audioPlayer.ts

```diff
@@ -0,0 +1,119 @@
+import { AVPlaybackStatus, Audio, InterruptionModeAndroid, InterruptionModeIOS } from 'expo-av';
+import { Sound } from 'expo-av/build/Audio/Sound';
+
+const mode = {
+	allowsRecordingIOS: false,
+	playsInSilentModeIOS: true,
+	staysActiveInBackground: true,
+	shouldDuckAndroid: true,
+	playThroughEarpieceAndroid: false,
+	interruptionModeIOS: InterruptionModeIOS.DoNotMix,
+	interruptionModeAndroid: InterruptionModeAndroid.DoNotMix
+};
+
+class AudioPlayer {
+	private audioQueue: { [uri: string]: Sound };
+	private audioPlaying: string;
+
+	constructor() {
+		this.audioQueue = {};
+		this.audioPlaying = '';
+	}
+
+	async loadAudio(uri: string): Promise<Sound> {
+		if (this.audioQueue[uri]) {
+			return this.audioQueue[uri];
+		}
+		const { sound } = await Audio.Sound.createAsync({ uri }, { androidImplementation: 'MediaPlayer' });
```

Comment:
What changes on this androidImplementation?

Reply:
https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/51756965550228597b4534fd50573a4ddfb0d4b5/app/containers/message/Audio.tsx#L306

https://docs.expo.dev/versions/latest/sdk/av/#avplaybacksource

support the same formats as Android's MediaPlayer

Reply:
I can see that. I meant why do you need this implementation over the default one.

Reply:
Because this implementation permits the Audio to reproduce .aac audios.